packages/a11y/src/A11y.spec.tsx

```diff
@@ -77,8 +77,15 @@ describe('packages/a11y', () => {
   let consoleSpy: jest.SpyInstance;
 
   describe('validateAriaLabelProps', () => {
-    beforeEach(() => (consoleSpy = jest.spyOn(console, 'error')));
+    beforeEach(
+      () =>
+        (consoleSpy = jest
+          .spyOn(console, 'error')
+          .mockImplementation(() => {})),
```

Comment:
What does adding this line fix?

Reply:
it mocks console.error so that we don't log the console errors to the terminal when we run them (less noise), while still giving us the ability to test that we are emitting console.errors when we want to